### ISSUE TITLE
opacity can not parse when is 84%

### ISSUE THREAD

**Issue by hhhenggg** [TylerBrinks/ExCSS#176]
![opacity](https://github.com/TylerBrinks/ExCSS/assets/45975545/8c3154ac-6665-4757-8f15-ff5af2863200)
When 84% cannot parse opacity. Unable to parse. When it is 0.84, it is correct

**Comment by cyraid** [comment]
I don't know the source well, and this is just from a quick browse but, it might be because:

```csharp
        private static readonly IValueConverter StyleConverter = Converters.NumberConverter.OrDefault(1f);
```

Might need a percentage and number converter there as per spec.

**Comment by TylerBrinks** [comment]
I will take a look at what's causing the issue.  That's a good place to start.

**Comment by TylerBrinks** [comment]
Fixed with the 2.4.6 release
